### PR TITLE
Make AAA style prominent

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

File: Snippets/Testing/Testing_6/Saga/SagaTests.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus.Testing;
+using NUnit.Framework;
+
+[TestFixture]
+public class SagaTests
+{
+    #region SagaTest [6.0,7.0)
```
Comment (SimonCropp) [inline Snippets/Testing/Testing_6/Saga/SagaTests.cs]: given this is in the `_6` dir the `[6.0,7.0)` is redundant

File: nservicebus/testing/fluent.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+reviewed: 2017-07-17
+component: Testing
+redirects:
+ - nservicebus/unit-testing
+ - nservicebus/testing/unit-testing
+related:
+ - samples/unit-testing
+---
+
+NOTE: Generally it is recommended to use Arrange-Act-Assert style tests. To learn how to test NServiceBus using `Arrange-Act-Assert (AAA)` approach, refer to the [dedicated article](/samples/unit-testing/).
```
Comment (SimonCropp) [inline nservicebus/testing/fluent.md]: i dont think Arrange-Act-Assert (AAA) should be code

File: nservicebus/testing/fluent.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+Testing handlers focuses on their externally visible behavior - the types of messages they send or reply with.
+
+snippet: TestingServiceLayer
+
+The test verifies that when a message of the type `RequestMessage` is processed by `MyHandler`, it responds with a message of the type `ResponseMessage`. Also, the test checks that if the request message's String property value is "hello" then that should be the value of the String property of the response message.
+
+
+## Sagas
+
+Testing sagas focuses on their externally visible behavior - the types of messages they send or reply with, but it's also possible to verify that saga requested a timeout or was completed.
```
Comment (SimonCropp) [inline nservicebus/testing/fluent.md]: so we recommend not to verify saga state?

Comment (danielmarbach) [inline nservicebus/testing/fluent.md]: I don't think we said that.

File: nservicebus/testing/fluent.md (hunk 2 above)
Comment (danielmarbach) [inline nservicebus/testing/fluent.md]: Will fix

File: Snippets/Testing/Testing_6/Saga/SagaTests.cs (hunk 1 above)
Comment (danielmarbach) [inline Snippets/Testing/Testing_6/Saga/SagaTests.cs]: Yeah will fix

File: nservicebus/testing/fluent.md (hunk 3 above)
Comment (SimonCropp) [inline nservicebus/testing/fluent.md]: was just wondering if it was implied.

**Comment by SimonCropp** [comment]
minor comments but LGTM

**Comment by danielmarbach** [comment]
Applied your comments @SimonCropp

**Pull request merged**